packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -58,30 +74,43 @@ export default class AssetGraphBuilder extends EventEmitter {
       entries
     });
 
+    this.handle = workerFarm.createReverseHandle(() => {
```

Comment:
I discovered a bug in the WorkerFarm. When Handles that are created in a worker are deserialized in the main process, what gets returned is actually a handle in the main process (https://github.com/parcel-bundler/parcel/blob/v2/packages/core/workers/src/Handle.js#L53). This problem was hidden before because there were 2 handles in the main process and 2 handles in the worker process. When I removed the `loadConfig` reverse handle I started getting the error `Error: Corresponding Handle was not found. It may have been disposed.` even though it was not being disposed. It was just being looked up where it didn't exist.